client/src/components/General/Signup/Component.jsx

```diff
@@ -0,0 +1,185 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Col, Row } from '../../../lib/Grid';
+import Button from '../../../lib/Button';
+import TextField from '../../../lib/TextField';
+import FieldError from '../../../lib/Feedback/FieldError';
+import { Margin } from '../../../Styles/utils';
+import { Logo, Brand } from '../../../Styles/bases/Identity';
+import { PATH_FORGOT_PASSWORD } from '../../../static/constants/appPath';
+import { Form, Field } from 'react-final-form';
+import {
+  dividerStyle,
+  FormHelperLink,
+  FullPageFilter,
+  FullPageLayer,
+  LoginContainer,
+  LoginTitleWrapper
+} from './styled';
+
+import {
+  composeValidators,
+  emailFormat,
+  emailRequired,
+  passwordFormat,
+  passwordLength,
+  passwordRequired
+} from '../../../utils/validationRules';
+
+import { Divider } from 'semantic-ui-react';
+class Signup extends Component {
+  state = {
+    signup: true,
+    emailAddress: '',
+    password: ''
```

Comment:
Another state for the confirm password field? See the problem[ here](https://gyazo.com/785a40a4c53ea794d846d37ddb1737be)

Reply:
Yea I will add another state for that. Thanks!

Reply:
Where did you update the state when we enter into the input fields? I tried to look for setState({...}) but I did not find any.